Cargo.toml

```diff
@@ -30,5 +30,5 @@ syn = "1.0"
 tempfile = "3.3.0"
 walrus = "0.23.3"
 wasmparser = "0.119.0"
-pocket-ic = "6.0.0"
+pocket-ic = "9.0.0"
```

Comment:
Should be `9.0.1`

Reply:
This is the library version. In crates.io, I don't see `9.0.1`, only [9.0.0](https://crates.io/crates/pocket-ic).

Reply:
oh sorry, my bad: 9.0.1 is the server version